### PR TITLE
HOCS-2670 Draft team Acceptance (MVP)

### PR DESCRIPTION
**Pull request by agraham19**
* Add new stage draft team acceptance stage

### CODE REVIEW

File: src/main/resources/processes/FOI_DRAFT_ACCEPTANCE.bpmn [hunk 1]
```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (SeanReedHomeOffice) [inline src/main/resources/processes/FOI_DRAFT_ACCEPTANCE.bpmn:1]: This stage should just be called FOI_ DRAFT or FOI_CONSIDER_AND_DRAFT

File: src/main/resources/processes/FOI_ACCEPTANCE.bpmn [hunk 2]
```diff
@@ -9,14 +9,15 @@
       <bpmn:outgoing>Flow_0eg5k1f</bpmn:outgoing>
```
Comment (SeanReedHomeOffice) [inline src/main/resources/processes/FOI_ACCEPTANCE.bpmn:9]: About ${execution.getVariable('DraftAcceptCase') != null && DraftAcceptCase == "N"}.
Not sure that DraftAcceptCase is the right variable to use - it is the output of a user task in a future stage. What we need to check is has this case already been accepted by the acceptance team. So I think maybe we should set a variable (date accepted?) in this stage when a case is accepted, and test for the presence of that variable.

**Review by SeanReedHomeOffice — COMMENTED** [review]
Looks good, a couple of comments

File: src/main/resources/processes/FOI_ACCEPTANCE.bpmn (hunk 2 above)
Comment (agraham19) [reply src/main/resources/processes/FOI_ACCEPTANCE.bpmn:9]: I have made this change

File: src/main/resources/processes/FOI_DRAFT_ACCEPTANCE.bpmn (hunk 1 above)
Comment (agraham19) [reply src/main/resources/processes/FOI_DRAFT_ACCEPTANCE.bpmn]: I have made this change

**Review by parth-ho — APPROVED** [review]

**Pull request merged**